pathway-analysis/ora_with_webgestaltr.Rmd

```diff
@@ -148,14 +152,15 @@ go_enrichment_results <-
               interestGeneType = "ensembl_gene_id",
               referenceGene = all_genes,
               referenceGeneType = "ensembl_gene_id",
-              outputDirectory = "results",
-              projectName = "GSE37418_up_SHH_lfc2_fdr0.05")
+              outputDirectory = results_dir,
+              projectName = "GSE37418_up_SHH_lfc2_fdr0_05" # Replace with a relevant output name for the HTML file
+              )
 ```
 
 As noted in the messages from `WebGestaltR`, the results are in
-`results/Project_GSE37418_up_SHH_lfc2_fdr0.05`.
+`results/Project_GSE37418_up_SHH_lfc2_fdr0_05`.
 The HTML report is at 
-[`results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0.05.html`](./results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0.05.html).
+[`results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0_05.html`](./results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0_05.html).
```

Comment:
```suggestion
[`results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0_05.html`](./results/Project_GSE37418_up_SHH_lfc2_fdr0.05/Report_GSE37418_up_SHH_lfc2_fdr0_05.html).
```

This file name looks odd to me (the part with the `fdr0_05`. But if that's what the file name is, I guess it's fine?

Reply:
Ahh both the directory and file name should end in `fdr0_05`, the reason being that it automatically is changed to this anyway when that chunk is ran, even when the file name ending in `fdr0.05` is given,